crates/tauri-bundler/src/bundle/linux/debian.rs

```diff
@@ -306,7 +306,7 @@ fn generate_md5sums(control_dir: &Path, data_dir: &Path) -> crate::Result<()> {
 /// Copy the bundle's resource files into an appropriate directory under the
 /// `data_dir`.
 fn copy_resource_files(settings: &Settings, data_dir: &Path) -> crate::Result<()> {
-  let resource_dir = data_dir.join("usr/lib").join(settings.main_binary_name());
+  let resource_dir = data_dir.join("usr/lib").join(settings.product_name());
```

Comment:
and also document the resource path changed - this will cause new apps to duplicate the resources in the system

Reply:
As far as I tested, at least `.deb` removes all bins and resources directory as well when installing a new version and then adds the new binaries and resources.

Reply:
> and also document the resource path changed

for tauri v1, the path is unchanged, it is only a breaking change for those on v2. Added a change file nevertheless